tests/mocks/function_gorest.py

```diff
@@ -0,0 +1,46 @@
+import json
+from typing import Any, Dict, List
+
+from requests import Response
+
+def mock_user_details_success(*args: List[Any], **kwargs: Dict[str, Any]):
+    """Mock for response success."""
+    result = {'id': 12345, 'name': 'Aishwarya', 'email': 'aisshwarya@test.com', 'gender': 'female', 'status': 'active'}     
+    
+    response = Response()
+    response.status_code = 200
+    response._content = bytes(json.dumps(result),encoding="utf-8")
+    
+    return response
+
+def mock_user_list_success(*args: List[Any], **kwargs: Dict[str, Any]):
+    """Mock for response success."""
+    result = [{'id': 12345, 'name': 'Aishwarya', 'email': 'aisshwarya@test.com', 'gender': 'female', 'status': 'active'}]
+    
```

Comment:
Okay. This is one way of testing api calls. To make it more relevant and efficient, we use vcr.py which will generate the mocks/response data that can be used for test cases.